components/form/appearance.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Appearance
+page_title: Form Appearance
+description: Appearance settings of the Form for Blazor.
+slug: form-appearance
+tags: telerik,blazor,form,appearance
+published: True
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the form by setting the [Size](#size) attribute.
+
+## Size
+
+You can increase or decrease the size of the Form by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Form.Size` class:
```

Comment:
Maybe you can add some more information that the Size controls the padding for all form editors and buttons.